tests/suites/test_suite_cmac.function

```diff
@@ -98,7 +98,7 @@ exit:
 }
 /* END_CASE */
 
-/* BEGIN_CASE */
+/* BEGIN_CASE depends_on:MBEDTLS_CIPHER_MODE_CBC */
```

Comment:
This test function doesn't look like it needs `MBEDTLS_CIPHER_MODE_CBC`. What's the problem? Does the implementation of CMAC depend on `MBEDTLS_CIPHER_MODE_CBC`? If so how come other test functions aren't affected?

It wouldn't be surprising that CMAC uses CBC code, since CMAC is basically a repurposed CBC. If so, the easiest fix would be to make `MBEDTLS_CMAC_C` depend on `MBEDTLS_CIPHER_MODE_CBC` in `check_config.h`.

Reply:
Done